src/Gateways/CominoGateway.cs

```diff
@@ -63,5 +71,22 @@ public class W2BatchPrintRow
             public string LetterType { get; set; }
             public string DocumentType { get; set; }
         }
+
+        public object GetDocumentConfig()
+        {
+            JObject config = JObject.Parse(
+                Environment.GetEnvironmentVariable("DOCUMENT_CONFIG")
+            );
+
+            JArray extractedCategories = (JArray)config.SelectToken("Categories");
+            JArray extractedDescriptions = (JArray)config.SelectToken("Descriptions");
+
+            object documentConfig = new {
```

Comment:
It might be nicer to define an actual type here, it should make accessing easier as well

Reply:
You can then just parse the JSON straight to that object like `JSONConvert.DesearialiseObject<YouTourHere>(EnvVar)` and return that